server/model/picture.js

```diff
@@ -9,9 +10,13 @@ export type Picture = {|
   alt?: ?string;
   copyright?: ?string;
   author?: string;
-  copyrightHolder?: string;
   fileFormat?: string;
   url?: ?string;
+  title?: ?string;
+  author?: ?string;
+  source?: ?{name?: ?string, link?: ?string};
+  license?: ?LicenseType;
+  copyright?: ?{holder?: string, link?: string};
```

Comment:
It'd be good to stop using double nullables...

I think having `license: ?LicenceType` is better, forcing someone to purposefully pass `null` if they don't have it. Just a thought for discussion.